kolibri/plugins/learn/assets/src/views/action-bar-search-box/index.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+
+  <div class="search-box-wrapper">
+    <div v-show="showSearchBox" :class="{ 'search-box-dropdown': showDropdownSearchBox }">
```

Comment:
I remember seeing some discussion behind this, but why are we tending to lean towards v-show now?